daemon/src/model/cfd.rs

```diff
@@ -2257,7 +2257,7 @@ mod tests {
                 Price::new(dec!(1000)).unwrap(),
                 Usd::new(dec!(100)),
                 Usd::new(dec!(1000)),
-                Origin::Theirs,
+                Origin::Ours,
```

Comment:
Note: This change was not strictly needed, but it is actually more correct, because we do test the `taker` with these... We should probably be more explicit once we create a more elaborate test framework...